src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -62,6 +62,10 @@ public static int Main(string[] args)
             {
                 return Log;
             });
+            new PackCommand(app, () =>
```

Comment:
After you fetch the latest code, you'll see that Push and Delete use the .Register pattern. So make a PackCommand.Register